medicinal_plants/lib/src/screens/collection-screen/widget/listTile_screen.dart

```diff
@@ -0,0 +1,34 @@
+// ignore_for_file: prefer_const_constructors, prefer_const_literals_to_create_immutables
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+import '../../../common/list-tile/list_tile.dart';
+import '../data/collection_screen_data.dart';
+
+class ListTileScreen extends StatelessWidget {
```

Comment:
your class names/widget names are confusing

for this case you can have it simply as collection_feed